docs/release_notes/4.4.26-25.md

```diff
@@ -0,0 +1,29 @@
+
+# Percona Server for MongoDB 4.4.26-25 (2023-10-16)
+
+[Installation](../install/index.md){.md-button}
+
+Percona Server for MongoDB 4.4.26-25 is a source available, highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 4.4.26 Community Edition enhanced [with enterprise-grade features](../comparison.md).
+
+It supports protocols and drivers of MongoDB 4.4.26.
+
+## Release Highlights
+
```

Comment:
* [AWS IAM authentication](../aws-iam.md) is now generally available, enabling you to use this functionality in production environments.

* Percona Server for MongoDB now includes telemetry that fills in the gaps in our understanding of how you use Percona Server for MongoDB to improve our products. Participation in the anonymous program is optional. You can opt-out if you prefer not to share this information. [Read more about Telemetry](../telemetry.md).